packages/next-on-pages/tests/templates/handleRequest.test.ts

```diff
@@ -1,7 +1,6 @@
 import { afterAll, expect, suite, test, vi } from 'vitest';
 import {
 	i18nTestSet,
-	infiniteLoopTestSet,
```

Comment:
Why is it not possible to create an e2e test for this?

Reply:
I think we simply haven't had a clear minimal reproduction for this (and I feel that trying to find/create one could be quite time consuming) 🤔 

I'll double check the i18n app as mentioned in #222 maybe I could use that to test the infinite loop 👍

Reply:
it doesn't look like the app is generating the issue (if the app mentioned in the issue is the one I think it is), so again I'm not sure if we can reproduce this, we could explore this but again I feel like the extra security around this rare edge case wouldn't be worth all the effort required to do that

@james-elicx, since you fixed the issue back when, do you have any idea of how it could be reproduced?

Reply:
i've been trying to reproduce this evening but had no luck 😭

Reply:
no problem, thanks a lot for giving it a try! 🙏

I feels like not testing it is not going to be a huge risk (especially since it's not even something that we can reproduce easily), hopefully you're ok with it?